shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -46,6 +47,12 @@
 @NotThreadSafe
 public class TfsShell implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static final HashMap<String, String[]> CMD_ALIAS = new HashMap<String, String[]>() { {
+      put("chgrpr", new String[] {"chgrp", "-R"});
```

Comment:
what is the motivation for having two ways of performing these recursive operations?